spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -126,29 +129,16 @@ ZipkinWebClientSender webClientSender(ZipkinProperties properties,
 	}
 
 	@Configuration(proxyBeanMethods = false)
-	static class ReporterConfiguration {
-
-		@Bean
-		@ConditionalOnMissingBean(Reporter.class)
-		@ConditionalOnBean(Sender.class)
-		AsyncReporter<Span> spanReporter(Sender sender, BytesEncoder<Span> encoder) {
-			return AsyncReporter.builder(sender).build(encoder);
-		}
-
-	}
-
-	@Configuration(proxyBeanMethods = false)
-	@ConditionalOnClass(ZipkinSpanHandler.class)
```

Comment:
ZipkinSpanHandler is a wrapper for those using core types via `Reporter<Span>`. It is more efficient and less deps to not force users through that, so removed that for `AsyncZipkinSpanHandler`